server/index.js

```diff
@@ -22,7 +22,7 @@ app.get('/:id', (req, res) => {
 
 app.get('/photos/:id', (req, res) => {
   const requestExpId = parseInt(req.params.id, 10);
-  db.getPhotos(requestExpId, (err, exData) => {
+  db.getExperiencePhotos(requestExpId, (err, exData) => {
     if (err) {
       res.status(400).send(err);
     } else {
```

Comment:
What happens if there is no error?

Reply:
github seems to have hidden line 29 which says "res.status(200).send(exData)", so I think it's OK